### PR TITLE
[Docs] Create a basic example of auto-logging with packager

### PR DESCRIPTION
**Pull request by moranbental**
Update the documentation for auto-logging dataset artifacts with package

### CODE REVIEW

File: docs/cheat-sheet.md [hunk 1]
```diff
@@ -590,6 +590,23 @@ log_with_returns_run = my_func.run(
 ```
 
 ### Automatic logging
+# Auto logging dataset basic example
+```python
+import pandas
+
+
+def func(col: list):
```
Comment (guy1992l) [inline docs/cheat-sheet.md]: and just for the good complete type hinted developer experience
```suggestion
def func(col: list) -> pd.DataFrame:
```

File: docs/cheat-sheet.md [hunk 2]
```diff
@@ -590,6 +590,23 @@ log_with_returns_run = my_func.run(
 ```
 
 ### Automatic logging
+# Auto logging dataset basic example
+```python
+import pandas
```
Comment (guy1992l) [inline docs/cheat-sheet.md]: even padnas don't import themselves as `padnas`
```suggestion
import pandas as pd
```

File: docs/cheat-sheet.md [hunk 3]
```diff
@@ -590,6 +590,23 @@ log_with_returns_run = my_func.run(
...
+
+
+def func(col: list):
+    df = pandas.DataFrame({"col": col})
+    return df
+
+
+function.run(
```
Comment (guy1992l) [inline docs/cheat-sheet.md]: function is a builtin in Python
```suggestion
my_func.run(
```

**Review by guy1992l — CHANGES_REQUESTED** [review]
I would add a second function that will receive the now logged DF via `inputs` and return a value to have a full cycle of auto-logging and parsing to show how to move objects between functions.

```python
func_2(df: pd.DataFrame) -> int:
    return int(df.sum()[0])
```

Then:
```python
run_1 = my_func.run(
    handler="func",
    params={"col": [1, 2, 3, 4, 5, 6, 7]},
    returns=["df:dataset"],
    local=True,
)
my_func.run(
    handler="func_2",
    inputs={"df": run_1["df"]},
    returns=["sum"],
    local=True,
)
```

**Review by guy1992l — APPROVED** [review]
Amazing :)

File: docs/cheat-sheet.md [hunk 4]
```diff
@@ -590,6 +590,34 @@ log_with_returns_run = my_func.run(
 ```
 
 ### Automatic logging
+# Auto logging dataset basic example
```
Comment (jillnogold) [inline docs/cheat-sheet.md]:
```python
# Auto-logging a dataset: basic example

Comment (guy1992l) [reply docs/cheat-sheet.md]: shouldn't it be a sub-sub-sub-sub-header? `####` instead of `#` if the line above has `###`?

**Review by guy1992l — APPROVED** [review]

**Review by jillnogold — APPROVED** [review]

**Review by alonmr — APPROVED** [review]

**Pull request merged**